src/run_atari_ppo.py

```diff
@@ -0,0 +1,67 @@
+"""Trains PPO agent and plays Atari Pong."""
+#!/usr/bin/env python
+
+import os.path as osp
+import logging
+from mpi4py import MPI
+import gym
+from baselines.common import set_global_seeds
+from baselines import bench
+from baselines import logger
+
+def wrap_train(env):
```

Comment:
this is redundant with `gen_pong_env`, can you move all env-wrapping code to `atari_env.py` for consistency?
 We should use `wrap_deepmind` (rather than our own hand-rolled `_wrap_deepmind` in `atari_env.py`, so this change should let you delete a bunch of our old code.